Improve readme and package dependency (#42)
taktik/ozone-components: packages/ozone-material/ozone-item-edit/Readme.md

 $ npm install --save ozone-item-edit
 ```

## ozone-item-edit

### usage

 ```html
<ozone-item-edit item-data={{item}} invalid="{{invalid}}">  </ozone-item-edit>
```
### attributes

**itemData**: GenericItem item to display.
**invalid**: boolean true if the value is invalid.

### methods

> **getUpdatedData**: get the item with it's modifies fields.

Reviewer: ```suggestion
**getUpdatedData**: get the item with its modified fields.
```

### Events:

> **value-changed**: Trigger when a value has changed

Reviewer: ```suggestion
**value-changed**: Triggered when a value has changed
```

 
 
 
## ozone-edit-xxx

### attributes

**type**: string ozone type of the entry
**value**: string value of the field
**name**: LocalizedString name of the field

> **language**: string anguage to use in LocalizedName

Reviewer: ```suggestion
**language**: string language to use in LocalizedName
```

**disabled**: boolean Set to true to disable this input.
**isModify**: boolean if the value is modify, is value will change to true.
**invalid**: boolean true if the value is invalid
**inputElement**: Returns a reference to the input element.

### events

>  **value-changed**: Trigger when a value has changed

Reviewer: ```suggestion
 **value-changed**: Triggered when a value has changed
```

>  **invalid-changed**: Trigger when invalid attribute has changed

Reviewer: ```suggestion
 **invalid-changed**: Triggered when invalid attribute has changed
```
